registration/views.py

```diff
@@ -81,6 +83,22 @@ def form_valid(self, form):
         verify_user_emails(user, user.email, self.object.public_contact_email)
         return response
 
+    # pylint: disable=arguments-differ
+    def get(self, *args, **kwargs):
+        """
+        Get registration form.
+
+        Retirects to new frontend if flag is set, otherwise
```

Comment:
Nit: typo:
```diff
- Retirects
+ Redirects
```

Reply:
Also we could add the name of the variable from settings (currently `REDIRECT_TO_NEW_CONSOLE`) to the docstring to be aware of this condition without reading the code.